systers_portal/templates/meetup/base.html

```diff
@@ -1,14 +1,14 @@
 {% extends "base.html" %}
 
 {% block title %}
-  {{ object.name }}
+  {{ meetup_location.name|capfirst }}
```

Comment:
I think it's okay to use capfirst, lets not leave any source of mistake. There is no harm in adding it.